### PR TITLE
Update tcpdf.php for #681

### PR DESCRIPTION
**Pull request by nshiddqui**
https://github.com/tecnickcom/TCPDF/issues/681

### CODE REVIEW

File: tcpdf.php [hunk 1]
```diff
@@ -6928,7 +6928,7 @@ protected function fitBlock($w, $h, $x, $y, $fitonpage=false) {
 	 * @public
```
Comment (williamdes) [inline tcpdf.php:6928]: Please add this after `@param array $altimgs`
```diff
+	 * @param int $padding
```

**Comment by nshiddqui** [comment]
i have added the params details over the function description. please approve it, so i can use it on my project ASAP.
Thanks

**Review by williamdes — APPROVED** [review]
Looks good to me
@nicolaasuni do we accept new parameters?

**Review by shaikh-usama — APPROVED** [review]

**Review by jabbarSoomro — APPROVED** [review]

**Comment by nicolaasuni** [comment]
This PR is introducing too many unrelated changes.

**Comment by nicolaasuni** [comment]
Please try to resolve the conflicts.

**Comment by nicolaasuni** [comment]
Hi, Thank you for your contribution.
Unfortunately at this moment I am not accepting new features as I am trying to complete tc-lib-pdf that is already a valid replacement of this lib, except for HTML/CSS/SVG/Javascript that are under development.

**Pull request closed without merging**